mmv1/products/networkservices/ServiceBinding.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceBinding'
+base_url: 'projects/{{project}}/locations/global/serviceBindings'
+create_url: 'projects/{{project}}/locations/global/serviceBindings?serviceBindingId={{name}}'
+self_link: 'projects/{{project}}/locations/global/serviceBindings/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
```

Comment:
Can you add a handwritten update test to make sure this works? There are examples of tests in https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests

Reply:
Thanks for catching this. Update on this resource is not supported. I fixed the yaml now.